frontend/main/src/components/shell/Sidebar.css

```
@@ -0,0 +1,292 @@
+.sidebar--collapsed .sidebar__user {
+  justify-content: center;
+  padding: var(--space-2);
+}
+.sidebar--collapsed .sidebar__nav-item {
+  justify-content: center;
+  padding: var(--space-2);
+  gap: 0;
+}
+
+.sidebar--collapsed .sidebar__nav-item--active::before {
+  top: 0;
```

Stylelint (17.14.0)

[error] Duplicate property "top" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)